courses/serializers.py

```diff
@@ -27,12 +29,23 @@ def get_programpage_url(self, program):
         except ProgramPage.DoesNotExist:
             return
 
+    def get_enrolled(self, program):
+        """
+        Returns true if the user
+        """
+        if 'request' in self.context:
+            user = self.context['request'].user
```

Comment:
No, `user` should be `AnonymousUser` if the user is not logged in. In any case we require the user to be logged in for the API using it